lib/srv/sess.go

```diff
@@ -987,15 +997,15 @@ func (s *session) startInteractive(ctx context.Context, ch ssh.Channel, scx *Ser
 	go func() {
 		result, err := s.term.Wait()
 		if err != nil {
-			scx.Errorf("Received error waiting for the interactive session %v to finish: %v.", s.id, err)
+			scx.WithError(err).Error("Received error waiting for the interactive session to finish")
 		}
 
 		// wait for copying from the pty to be complete or a timeout before
 		// broadcasting the result (which will close the pty) if it has not been
 		// closed already.
 		select {
 		case <-time.After(defaults.WaitCopyTimeout):
-			s.log.Errorf("Timed out waiting for PTY copy to finish, session data for %v may be missing.", s.id)
+			s.log.Error("Timed out waiting for PTY copy to finish, session data  may be missing.")
```

Comment:
Is there any reason `s.log` is used here instead of the `*logrus.Entry` embedded in `scx` as above?